### PR TITLE
added focus-areas template

### PR DESCRIPTION
**Pull request by ritik-malik**
Hi,

As discussed in #276, I added the templates for focus-areas READMEs,
Would be happy to do the suggested changes.

Signed-off-by: ritik-malik <ritik18406@iiitd.ac.in>

### CODE REVIEW

File: templates/focus-areas.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+This  file is divided into 2 parts:
+1. Template for `WG/focus-areas/README.md`
+2. Template for `WG/focus-areas/<focus-area-name>/README.md`
+
+## Part 1
+
+The following template describes the structure of READMEs for focus-areas in each WG repositories:
+
+---
```
Comment (GeorgLink) [inline templates/focus-areas.md:11]: Instead of lines, use 
\`\`\`

That will make the template look more like a template and can be copied more easily.

Comment (ritik-malik) [reply templates/focus-areas.md]: Done

**Comment by GeorgLink** [comment]
Thanks for making the change. It looks great.

One more suggestion: change the headings "Part 1" and "Part 2" to something more descriptive.

**Comment by ritik-malik** [comment]
hmm how about we use their entire name as written in line 4 and 5 -

`## Template for WG/focus-areas/README.md` instead of `## Part 1`

and 

`## Template for WG/focus-areas/<focus-area-name>/README.md` instead of `## Part 2`

**Comment by GeorgLink** [comment]
Good idea

**Comment by GeorgLink** [comment]
Thank you !

**Pull request merged**